lib/vpn/vpn_tab.dart

```diff
@@ -1,17 +1,22 @@
 import 'package:lantern/account/split_tunneling.dart';
 import 'package:lantern/messaging/messaging.dart';
 import 'package:lantern/vpn/vpn.dart';
+import 'package:lantern/vpn/vpn_notifier.dart';
+import 'package:shimmer/shimmer.dart';
+
+import '../common/ui/custom/internet_checker.dart';
 import 'vpn_bandwidth.dart';
 import 'vpn_pro_banner.dart';
 import 'vpn_server_location.dart';
 import 'vpn_status.dart';
 import 'vpn_switch.dart';
 
 class VPNTab extends StatelessWidget {
-  VPNTab({Key? key}) : super(key: key);
+  const VPNTab({Key? key}) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    final vpnModel = context.watch<VPNChangeNotifier>();
```

Comment:
Maybe rename this to vpnNotifier? I initially thought we were calling isFlashlightInitialized on this [vpnModel](https://github.com/getlantern/lantern-client/blob/main/lib/vpn/vpn_model.dart)

Reply:
Changing....